specs/Standards/NonFungibleToken/payouts.md

```diff
@@ -0,0 +1,58 @@
+# Standard for a Multiple-Recipient-Payout mechanic on NFT Contracts
+
+A standard for supporting structures that allow for NFT's to inform financial
+contracts to payout multiple addresses on sales.
+
+Currently, NFT's on Near support the field `owner_id`, but lack flexibility for
+ownership and payout mechanics with more complexity, including but not limited
+to royalties. Financial contracts, such as Marketplaces, Auction-houses, and
+NFT Loan contracts, would benefit from a standard interface on NFT producer
+contracts for querying whom to pay out, and how much to pay.
+
+Therefore, the core goal of this standard is to define a set of methods for
+financial standard set of methods to call, without specifying how NFT contracts
+define the divide of payout mechanics.
+
+The `Payout` interface is should be a simple collection of unique addresses that
+map to either a proportional ownership of a Token, or even simpler, a mapping of
+unique addresses to the Balances each is to receive, given an initial balance:
+```rust
+    // return fractional ownership
+    pub struct OwnershipFractions {
+      pub fractions: HashMap<AccountId, MultipliedSafeFraction>
```

Comment:
I like your suggestion to go with the latter. I think in a proposal of a new standard, you could just propose your best guess in the main PR, and offer alternatives as PRs targeting the main PR. Or in comments, if a second PR seems like overkill or is hard to pull off for non-maintainers of the repo.